expressions.go

```diff
@@ -319,6 +319,36 @@ var functions = map[string]struct {
 			return nil, fmt.Errorf("sum expects argument #1 to be slice of numbers")
 		}
 	}},
+	"sumiftrue": {2, func(args []Value) (Value, error) {
```

Comment:
Based on convo in slack, let's rename from `sumiftrue` to `sumif` to match Excel and Google Sheet.s

Reply:
with the `sumif` found in Excel & Google Sheets, you can specify the condition. Ideally we'd get get there eventually, but I don't think that would be easy to support right now. IMO it would be misleading to name this `sumif` since the only condition it supports right now is that the element in the 2nd slice is `true`